README.md

```diff
@@ -1,6 +1,32 @@
 # Product-Portal
 
-## Installation
+## Project Details
+
+### How do we complete this project?
+
+#### Ver 0.1 :
+  - **General concept:** Create a static website using array of objects (30 items - will be made by xx and yy) (each json item: title,price,link,catgeory…)
+    - navbar will include 6~ categories (which will be connected with the json file to right product category)
+    - Search bar input above / beside the navbar (see mockup)
+    - Home/index page will display random 16 items each time with (back - next page at the bottom)
+    - Each li product will display item title > picture > price > button (one below the other in this order)
```

Comment:
What does clicking on the product do?  Does it redirect the user to the actually product page somewhere else?  Is it going to try to show more info about the product?  Or does our website just display the info and does nothing else?  The ver 1.0 says we'll have baskets, but right now It's unclear on what clicking on the product does.